tests/test_models.py

```diff
@@ -85,7 +85,16 @@ def test_model_singlepsr_noise_dip_cusp(nodmx_psrs,caplog):
 
 def test_model_singlepsr_noise_chrom_nondiag(nodmx_psrs,caplog):
     # caplog.set_level(logging.CRITICAL)
-    m=models.model_singlepsr_noise(nodmx_psrs[1], chrom_gp=True)
+    m=models.model_singlepsr_noise(nodmx_psrs[1], dm_var=True,
+                                   dm_type=None, chrom_gp=True,
+                                   chrom_gp_kernel='non_diag')
```

Comment:
I think the correct flag is `'nondiag'`, see L96 below

Reply:
ok just made another commit